dll/win32/shell32/brfolder.cpp

```diff
@@ -1034,6 +1113,14 @@ BrFolderDlgProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
     if (!info)
         return 0;
 
+    if (info->pContextMenu)
+    {
+        LRESULT result;
+        if (S_OK == SHForwardContextMenuMsg(info->pContextMenu, uMsg, wParam,
+                                              lParam, &result, TRUE))
```

Comment:
```suggestion
                                            lParam, &result, TRUE))
```
indentation was off. Maybe it even makes sense to put curly braces around the return-part here, because the condition of the if is multiline. It didn't want to use the suggestion-feature here to not make Katayamas comment with the `TRUE or` outdated. That looks like a more relevant bug.

Reply:
Also we don't want yoda-style code, so it should be:
`if (function() == S_OK)`.
In addition, per https://reactos.org/wiki/Coding_Style#Braces , since your if-condition is more than 1 line long, the if-body should have braces.